android/src/main/java/nordpol/AndroidCard.java

```diff
@@ -116,4 +137,9 @@ public void close() throws IOException {
     public Tag getTag() {
         return card.getTag();
     }
+    
+    public byte[] getHistoricalBytes() {
```

Comment:
We intend to use Nordpol core also outside of Android that is why we opted for not having getTag in core.